src/org/labkey/test/components/react/TemplateDownloadButton.java

```diff
@@ -0,0 +1,122 @@
+package org.labkey.test.components.react;
+
+import org.labkey.test.Locator;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.io.File;
+
+import static org.labkey.test.WebDriverWrapper.WAIT_FOR_JAVASCRIPT;
+import static org.labkey.test.WebDriverWrapper.waitFor;
+
+public class TemplateDownloadButton extends MultiMenu
+{
+    private static final String DEFAULT_TEMPLATE_NAME = "Default Template";
+    private static final String BUTTON_CLASS = "template-download-button";
+    public static final Locator LOCATOR = Locator.tagWithClass("button", BUTTON_CLASS);
+
+    public TemplateDownloadButton(WebElement element, WebDriver driver)
+    {
+        super(element, driver);
+    }
+
+    private File download(String templateName)
+    {
+        return getWrapper().doAndWaitForDownload(() -> {
+            clickAndWaitForLoaded();
```

Comment:
Make it explicit here that this might be what triggers the download.
```suggestion
            clickAndWaitForLoaded(); // Will trigger download if there are no custom templates
```